Source/WebCore/Modules/reporting/ReportingScope.cpp

```diff
@@ -181,7 +181,7 @@ void ReportingScope::generateTestReport(String&& message, String&& group)
         if (group.isNull())
             group = "default"_s;
         
-        document->sendReportToEndpoints(testReportURL, { }, { group }, WTFMove(reportFormData), ViolationReportType::Test);
+        document->sendReportToEndpoints(testReportURL, { }, singleElementSpan(group), WTFMove(reportFormData), ViolationReportType::Test);
```

Comment:
@brentfulgham Not just where you pointed to, this places avoids a Vector construction and there are a few others in this PR (see calls to singleElementSpan).